docs/content/concepts/sui-move-concepts/move-on-sui.mdx

```diff
@@ -28,33 +28,30 @@ Key differences with Move on Sui include:
 
 ### Object-centric global storage {#global-storage}
 
-In Diem-style Move, global storage is part of the programming model that you can access through special operations, such as `move_to`, `move_from`, and more global storage operators. Core Move stores both resources and modules in global storage. When you publish a module, it's stored into a newly generated module address inside Move. When you create a new object (resource), it's usually stored into some address, as well. This approach creates a problem, however, as on-chain storage is expensive and limited (not optimized for storage and indexing). Current blockchains cannot scale effectively to handle storage-heavy applications, such as marketplaces and social apps.
```

Comment:
@sblackshear, @tnowacki, this passage didn't really match my understanding of how addressing/resources/modules worked on Diem, and what the scaling pitfalls were with that, but I wasn't around at the time -- could you check my working here?